cloud/keystone.go

```diff
@@ -91,6 +96,8 @@ func (client *keystoneV3Client) VerifyToken(token string) (schema.Authorization,
 
 	tokenResult := v3tokens.Get(client.client, token)
 	if tokenResult.Err != nil {
+		// tokenResult.Err.Error() is stripped out of useful info
+		log.Warning("Error during verifying token: %#v", tokenResult.Err)
```

Comment:
We could also return `%#v` to the user but I'm afraid that it may expose too many details. WDYT?

Reply:
Agree that returning this in error exposes too much details, so better to keep it as log.
Why not just `+v` instead of `#v`?

Reply:
I'm fine with returning a terse error and having a detailed log message. But I would log this on Error level.

Reply:
@jkmar `+v` prints result of `.Error()` instead of the struct fields. See https://play.golang.org/p/wE0OWAA9cYS

Reply:
@cierpuchaw I agree, changed to `Error`.